processor/src/tasks/processKicadPCB/index.ts

```diff
@@ -46,7 +46,13 @@ async function processKicadPCB(
         job.updateProgress({ status: 'failed', file: layoutSvgPath, error }),
       )
 
-    const gerbers = await gerbersPromise
+    // Only plot gerbers if there's no `gerbers` key in kitspace.yaml.
+    let gerbers = []
+    if (kitspaceYaml.gerbers == null) {
+      gerbers = await gerbersPromise
+    }
+
+    // const gerbers = await gerbersPromise
```

Comment:
This way it's still plotting the gerbers but not using the result so it's wasted work. (Also, there's some commented out code which should be removed.)